third_party/zap/templates/client-command-macro.zapt

```diff
@@ -0,0 +1,3 @@
+{{chip_header}}
+
+// TODO issue #3637
```

Comment:
Since this file is unused at the moment, maybe we don't even need to include it ?